content/docs/user-guide/experiment-management/comparing-experiments.md

```diff
@@ -0,0 +1,507 @@
+        Unique values:         5
+        Smallest value:        0.9127
+        Largest value:         0.9167
+        Sum:                   5.4895
+        Mean:                  0.914917
+        Median:                0.91565
+        StDev:                 0.001774
+        Most common values:    0.9127 (2x)
+                               0.9167 (1x)
+                               0.9153 (1x)
+                               0.9161 (1x)
+                               0.916 (1x)
+...
+```
+
+[csvkit]: https://csvkit.readthedocs.io/en/latest/
+
+## Compare specific experiments
+
+In addition to showing a summary table of experiments, DVC provides a command to
+compare pairwise experiments. `dvc exp diff` is used to compare the experiments
+by the change in their metrics and params.
+
+```dvc
+$ dvc exp diff
+Path          Metric    Value    Change
+metrics.json  acc       0.9151   0.0024
+metrics.json  loss      0.23867  0.0020977
+
+Path         Param             Value    Change
+params.yaml  model.conv_units  256      240
+```
+
+Without experiment names, `dvc exp diff` shows the change between the last
+experiment and the last commit. So if the command doesn't print an output, there
+might be no experiment since the previous commit or it didn't produce changes in
+results. If you want to see all the parameters and metrics regardless of whether
+they have changed, you can use `--all` flag.
+
+```dvc
+$ dvc exp diff --all
+Path          Metric    Value    Change
+metrics.json  acc       0.9151   0.0024
+metrics.json  loss      0.23867  0.0020977
+
+Path         Param             Value    Change
+params.yaml  model.conv_units  256      240
+params.yaml  train.epochs      10       0
+```
+
+## Compare an experiment with the workspace
+
+When you want to compare two experiments, either the baseline experiment in a
+commit, branch, tag or an attached experiment with ID, you can supply their
+names to `dvc exp diff`.
+
+```
+$ dvc exp diff exp-25a26 cnn-64
+Path          Metric    Value    Change
+metrics.json  acc       0.9153   0.00020003
+metrics.json  loss      0.23385  -0.0048174
+
+Path         Param             Value    Change
+params.yaml  model.conv_units  64       -192
+```
+
+## Customize the comparison output
+
+You can hide the file path of metrics and parameter files the `--no-path` option
+-- produces cleaner output for single params/metrics files, for example. You can
+also customize the amount of significant digits shown for numeric values with
+the `--precision` option (5 by default).
+
+```dvc
+$ dvc exp diff exp-25a26 cnn-64 --no-path --precision 2
+Metric    Value    Change
+acc       0.92     0.0002
+loss      0.23     -0.0048
+
+Param             Value    Change
+model.conv_units  64       -192
+```
+
+### Get the comparison in JSON
+
+Parsing the `dvc exp diff` output may not be feasible due to the custom table
+layout. When you want to use the output in other commands, `dvc exp diff` can
```

Comment:
💅 "custom" sounds like user customized something but we're referring to the output of `exp diff` in general.
```suggestion
Parsing the output of `dvc exp diff` may not be easy when
you want to use it in other commands. `dvc exp diff` can
```